index.html

```diff
@@ -46,7 +54,8 @@ <h2 class="headline-section__sub-title">
 		<section class="work-section">
 			<div class="work-section__box">
 			<div class="work-section__card">
-				<div class="work-section__card--img"><img src="/images/SnapshootPortfolio.png" alt=""></div>
+				<div class="work-section__card--img mobile"><img src="/images/SnapshootPortfolio.png" alt="card snap"></div>
+				<div class="work-section__card--img desktop"><img src="./images/Card-images/SnapshootPortfolioDesktop1.png" alt="card snap"></div>
```

Comment:
- [ ] Kindly please correct these elements so their alignment matches the description and the project title, like in the Figma:

![image](https://user-images.githubusercontent.com/57421823/149451291-27839163-6552-4269-abb3-93174f53f4b2.png)

- [ ] Please kindly add the project titles that are missing too.

Let's do this so the projects featured in your portfolio look more striking! 💪